### PR TITLE
samba: fix avahi propagation

### PR DESCRIPTION
**Pull request by chincheta0815**
As I am struggling with samba-4.10.0 it would be great to have that patch in.

So here is the original PR I closed since I thought samba-4.10 would be easy...: https://github.com/OpenIndiana/oi-userland/pull/4901

### CODE REVIEW

File: components/network/samba/manifests/sample-manifest.p5m [hunk 1]
```diff
@@ -1,27 +1,3 @@
-#
```
Comment (pyhalov) [inline components/network/samba/manifests/sample-manifest.p5m]: Don't touch this file, it didn't change.

File: components/network/samba/samba.proto-objects [hunk 2]
```diff
@@ -1,10 +1,4 @@
-dir  path=etc
```
Comment (pyhalov) [inline components/network/samba/samba.proto-objects]: The same goes for this one.

File: components/network/samba/patches/11-avahi-register.patch [hunk 3]
```diff
@@ -0,0 +1,20 @@
+--- samba-4.9.5/source3/smbd/avahi_register.c	2019-03-12 09:19:04.000000000 +0000
```
Comment (pyhalov) [inline components/network/samba/patches/11-avahi-register.patch:6]: Please, add reference to https://www.illumos.org/issues/10577 on the top of this patch.

**Comment by pyhalov** [comment]
Commit message should be "samba: fix avahi propagation"

File: components/network/samba/manifests/sample-manifest.p5m (hunk 1 above)
Comment (chincheta0815) [reply components/network/samba/manifests/sample-manifest.p5m]: Thanks. I wondered why it should have changed, but this time I at least did not forget to create that manifest... ;o)

File: components/network/samba/samba.proto-objects (hunk 2 above)
Comment (chincheta0815) [reply components/network/samba/samba.proto-objects]: Same as above.

File: components/network/samba/patches/11-avahi-register.patch (hunk 3 above)
Comment (chincheta0815) [reply components/network/samba/patches/11-avahi-register.patch:6]: Will do that.

**Comment by chincheta0815** [comment]
Will do the changes and then quash and push...

**Comment by chincheta0815** [comment]
@pyhalov I hope I did everything well. Concerning samba-4.10: Could you provide me some help or "infopoint"? Samba-devs seem to have different topics on their agenda....

**Pull request merged**